diff --git a/index.php b/index.php
index b7f2629..a84caf9 100644
--- a/index.php
+++ b/index.php
@@ -2,8 +2,14 @@
 
 class Ship {
     public $name;
-
+    public $weaponPower;
+    public $jediFactor;
+    public $strength;
 }
+
+$myShip = new Ship();
+$myShip->weaponPower = 100;
+
 ?>
 
-<span>Look! My ship has <strong><?php ?></strong> weapon power</span>
+<span>Look! My ship has <strong><?php echo $myShip->weaponPower; ?></strong> weapon power</span>
