diff --git a/index.php b/index.php
index 4f0cff7..b7f2629 100644
--- a/index.php
+++ b/index.php
@@ -1,16 +1,9 @@
 <?php
-require 'functions.php';
 
-$ships = get_ships();
+class Ship {
+    public $name;
 
-$currentStrength = 0;
-$strongestShip = [];
-foreach ($ships as $ship) {
-    if ($ship['strength'] > $currentStrength) {
-        $strongestShip = $ship;
-        $currentStrength = $ship['strength'];
-    }
 }
 ?>
 
-<strong>The strongest ship is: </strong><?php echo $strongestShip['name'];?>
+<span>Look! My ship has <strong><?php ?></strong> weapon power</span>
