diff --git a/index.php b/index.php
index 247f39e..4f0cff7 100644
--- a/index.php
+++ b/index.php
@@ -1,6 +1,16 @@
 <?php
 require 'functions.php';
 
+$ships = get_ships();
 
+$currentStrength = 0;
+$strongestShip = [];
+foreach ($ships as $ship) {
+    if ($ship['strength'] > $currentStrength) {
+        $strongestShip = $ship;
+        $currentStrength = $ship['strength'];
+    }
+}
 ?>
-<strong>The strongest ship is: </strong>
+
+<strong>The strongest ship is: </strong><?php echo $strongestShip['name'];?>
